commands/files/serialfile.go

```diff
@@ -22,50 +17,34 @@ type serialFile struct {
 	path    string
 	files   []os.FileInfo
 	stat    os.FileInfo
-	current *os.File
+	current *File
 }
 
 func NewSerialFile(name, path string, stat os.FileInfo) (File, error) {
-	if stat.Mode()&os.ModeSymlink != 0 {
-		target, err := os.Readlink(path)
+	switch mode := stat.Mode(); {
+	case mode.IsRegular():
+		file, err := os.Open(path)
 		if err != nil {
 			return nil, err
 		}
-
-		return NewLinkFile("", path, target, stat), nil
```

Comment:
probably for no good reason, although I don't believe it actually gets used (same with the passed in `stat`) it was just for consistency across file constructors